appinventor/appengine/src/com/google/appinventor/client/utils/HTML5DragDrop.java

```diff
@@ -108,6 +109,13 @@ private static native void initJsni()/*-{
       $entry(@com.google.appinventor.client.utils.HTML5DragDrop::checkProjectNameForCollision(*));
     top.HTML5DragDrop_shouldShowDropTarget =
       $entry(@com.google.appinventor.client.utils.HTML5DragDrop::shouldShowDropTarget(*));
+    top.importProjectFromUrl =
+      $entry(@com.google.appinventor.client.utils.HTML5DragDrop::importProjectFromUrl(*));
```

Comment:
I'm not clear why this was added. It doesn't seem to be called from anywhere.

Reply:
right, rather`top.HTML5DragDrop_importProject = importProject;` was needed in html5dnd.js

Reply:
I'm not sure I understand your comment. The general logic of the `initJsni` function is that it exposes the GWT functions to JavaScript. This line of code defines the JS name `importProjectFromUrl`, but none of the JS code calls this function. I believe this line could be removed since the function is unused from the JS side.

Reply:
yes, I removed that line. I needed to expose the function in html5dnd.js, not here